scan.c

```diff
@@ -97,11 +102,17 @@ int corsairlink_device_scanner(libusb_context *context)
 						scanlist_count++;
 						break;
 					} else {
+						msg_debug("No (device_id 0x%02X)\n", device_id);
+						
 						corsairlink_handle_close(scanlist[scanlist_count].handle);
 						continue;
 					}
 				}
-			}
+				else
+				{
```

Comment:
Curly brackets should be on the same line as the else.

Reply:
hmmm... I guess vscode misaligned that...